ctk/idp/coverage/NotTested.md

```diff
@@ -0,0 +1,59 @@
+| 2.7.4 - Element <AuthzDecisionStatement> | In order for the assertion to be interpreted correctly and securely, the SAML authority and SAML relying party MUST interpret each URI reference in a consistent manner. | Impossible to verify how the SAML authority interprets the URI it provided.
+| 3.2.2 - Complex Type StatusResponseType | If such a signature is used, then the \<ds:Signature> element MUST be present. | Impossible to directly verify since it's impossible to know "If such a signature is used". However the \<ds:Signature> element is tested if provided.
+| 3.3.4 - Processing Rules | If S2 includes one or more \<saml:SubjectConfirmation> elements, then S1 MUST include at least one \<SAML:SubjectConfirmation> element such that S1 can be confirmed in the manner described by at least one \<SAML:SubjectConfirmation> element in S2. | "Confirming" in \<SAML:SubjectConfirmation> is a very difficult task. For now only the \<SAML:SubjectConfirmation> Method attributes are compared, rather than actual confirmation.
+| 3.4.1 - Element <AuthnRequest> | If \[ForceAuthn is\] "true", the identity provider MUST authenticate the presenter directly rather than rely on a previous security context. If a value is not provided, the default is "false". | Hard to test because how they use “previous security context” is implementation-specific. We could catch most cases by handling cookies and/or session variables, for example, but it wouldn’t cover all cases.
+| 3.4.1 - Element <AuthnRequest> | If \[isPassive is\] "true", the identity provider and the user agent itself MUST NOT visibly take control of the user interface from the requester and interact with the presenter in a noticeable fashion. | Impossible to test because the tests cannot reliably tell if an HTML response is going to display anything in the browser. (for example, javascript could be embedded in the HTML that dynamically changes the HTML).
+| 3.4.1 - Element <AuthnRequest> | If both ForceAuthn and IsPassive are "true", the identity provider MUST NOT freshly authenticate the presenter unless the constraints of IsPassive can be met. | Impossible to test. See IsPassive.
+| 3.4.1 - Element <AuthnRequest> | AssertionConsumerServiceIndex usage | Can't be tested until a second binding (Artifact) is supported in the test kit, since we won't know if they're just relying on the default.
+| 3.4.1 - Element <AuthnRequest> | The identity provider MUST have a trusted means to map the \[AssertionConsumerServiceIndex\] index value in the attribute to a location associated with the requester. \[SAMLMeta\] provides one possible mechanism. | Hard to test because we don’t know what method the IdP is going to use.
```

Comment:
❓ Isn't the reason above
```
Can't be tested until a second binding (Artifact) is supported in the test kit, 
since we won't know if they're just relying on the default.
```
why we can't test this.